Doc/library/stdtypes.rst

```diff
@@ -1817,9 +1817,15 @@ expression support in the :mod:`re` module).
    See :ref:`error-handlers` for details.
 
    For performance reasons, the value of *errors* is not checked for validity
-   unless an encoding error actually occurs,
-   :ref:`devmode` is enabled
-   or a :ref:`debug build <debug-build>` is used.
+   unless an encoding error actually occurs, :ref:`devmode` is enabled or a :ref:`debug build <debug-build>` is used.
```

Comment:
Let's keep the original whitespace for this unchanged text.

We generally leave unchanged lines, and in any case try and limit to 80 chars.

https://devguide.python.org/documentation/markup/#use-of-whitespace

```suggestion
   unless an encoding error actually occurs,
   :ref:`devmode` is enabled
   or a :ref:`debug build <debug-build>` is used.
```